components/event-processor/org.wso2.carbon.event.processor.admin/pom.xml

```diff
@@ -95,6 +95,7 @@
                             <!--!javax.xml.namespace,-->
                             org.wso2.carbon.event.processor.core.*,
                             javax.xml.namespace; version=0.0.0,
+                            org.wso2.carbon.databridge.*;version="${carbon.analytics.common.range}",
```

Comment:
@ayyoob 
Since the dependency is on the databridge commons, we don't need to import all the databridge packages. Importing databridge.commons and its subpackages would be enough. Can you adjust the import accordingly?